api/models/auxil/constants.py

```diff
@@ -71,6 +71,8 @@ class GroupDiscardReason(str, Enum):
     )
 }
 
+TIME_SLOTS = [(5, 8), (8, 11), (11, 14), (14, 17), (17, 21)]
+
```

Comment:
I see, we're using it in two modules so we need a constant, fine. Please move it down, under `TEACHER_PEER_SUPPORT_OPTIONS`. The rest down there is not sorted alphabetically, don't bother, I guess :)